subsys/net/conn_mgr: set a thread name

laroche opened the pull request:
Having a thread name might make it easier to know what this thread is all about.
E.g. from the list of threads within the shell.

best regards,

Florian La Roche

rlubos reviewed (commented):
Plese use `net: conn_mgr: ` prefix in the commit title (no subsys). And all commits need some description in the body, even simple ones.

laroche commented on subsys/net/conn_mgr/conn_mgr_monitor.c:
```diff
@@ -338,6 +338,9 @@ static int conn_mgr_mon_init(void)
 			CONFIG_NET_CONNECTION_MANAGER_MONITOR_STACK_SIZE,
 			conn_mgr_mon_thread_fn,
 			NULL, NULL, NULL, THREAD_PRIORITY, 0, K_NO_WAIT);
+	if (IS_ENABLED(CONFIG_THREAD_NAME)) {
```
Right. I've seen both ways within the source. As this might not end up in production code, I could remove it.

Should I send in a new patch without this "if()" line?

laroche committed:
net: conn_mgr: set a thread name

Set a thread name for net conn_mgr, so that e.g. thread listings
within the shell are easier to look at.

Signed-off-by: Florian La Roche <Florian.LaRoche@gmail.com>
